app/com/linkedin/drelephant/tuning/obt/AlgorithmManagerOBT.java

```diff
@@ -34,48 +25,52 @@
 import models.TuningAlgorithm;
 import models.TuningJobDefinition;
 import models.TuningParameter;
-import models.TuningParameterConstraint;
 import org.apache.commons.io.FileUtils;
 import org.apache.log4j.Logger;
 import play.libs.Json;
 import org.apache.hadoop.conf.Configuration;
 
+public class AlgorithmManagerOBT extends AbstractAlgorithmManager {
```

Comment:
The abstraction with regards to the algorithm isn't perfect here. The code related to PSO/IPSO should go in a separate class. This could be the driver class which in turn invokes algorithm specific methods.